components/multi-platform-controller/production-downstream/replicas_patch.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: multi-platform-controller
   namespace: multi-platform-controller
 spec:
-  replicas: 30
+  replicas: 1
```

Comment:
if we reduce to 1, do we need to remove the pod affinity section, line 8 to 21?

Reply:
we''ll remove it later after experiments